src/org/loklak/Caretaker.java

```diff
@@ -77,7 +77,10 @@ public void run() {
         Thread.currentThread().setName("CARETAKER");
         // send a message to other peers that I am alive
         String[] remote = DAO.getConfig("backend", new String[0], ",");
-        
+
+        int maxRetries = Integer.valueOf(DAO.getConfig("caretaker.backendpush.retries", "5"));
+        int backOffParameter = Integer.valueOf(DAO.getConfig("caretaker.backendpush.backoff", "3000"));
```

Comment:
They are set to default values here. If anyone wishes to change these values, they may simply define it in `config.properties`.